enterprise/service/license.go

```diff
@@ -108,21 +98,37 @@ func (s *licenseService) parseClaims(claims jwt.MapClaims) (*enterpriseAPI.Licen
 	if !ok {
 		return nil, fmt.Errorf("plan is not valid, found '%v'", claims["plan"])
 	}
-	if err := validPlanType(plan); err != nil {
-		return nil, err
+
+	planType, err := convertPlanType(plan)
+	if err != nil {
+		return nil, fmt.Errorf("plan type %q is not valid", planType)
 	}
 
 	aud, ok := claims["aud"].(string)
 	if !ok || aud == "" {
 		return nil, fmt.Errorf("aud is not valid, found '%v'", claims["aud"])
 	}
 
-	return &enterpriseAPI.License{
+	license := &enterpriseAPI.License{
 		InstanceCount: instance,
 		ExpiresTs:     exp,
-		Plan:          plan,
+		Plan:          planType,
 		Audience:      aud,
-	}, nil
+	}
+
+	if err := license.Valid(); err != nil {
+		return nil, err
+	}
+
+	s.l.Info(
+		"parse license succeed",
+		zap.String("plan", license.Plan.String()),
+		zap.Time("expiresAt", time.Unix(license.ExpiresTs, 0)),
+		zap.String("audience", license.Audience),
```

Comment:
We may not need to print audience since it's always the same?

Reply:
I suggest to move the log to LoadLicense so that we keep the error / non-error log in the same place.